docs/resources/covid-19.md

```diff
@@ -1,44 +1,182 @@
 # Covid-19 and Travel Safety 
 
-Access travel Restriction data for over 200 countries
 
-[Keep Travelers informed with Amadeus Travel Restriction API ](https://developers.amadeus.com/blog/-introducing-amadeus-covid-19-travel-restrictions-api)
 
-Powered by [Riskline](https://riskline.com/), the travel restrictions data is updated every hour to so you can keep your travelers informed of the latest developments on their trip. 
+"Can I travel to South Korea? Thailand? or Costa Rica? Do I need to have a PCR test? before the flight? after I arrive? How many days of quarantines are required? Where can I find this information?" 
+
+Don't worry, with Amadeus Self Service APIs, you will be able to have the details information of **current restrictions, Number of covid-19 cases, Entry requirements, and Safety scores, etc** with more details in a searched destinations with one single API.
+
+and even more, **safety information** can be retrieved for the destination you would like to visit so that you will be aware of where you are heading to. 
 
 ## Search by an area
-the [Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) can be searched by a country, city or region. 
- 
+
+There are 2 APIs to achieve the information 
+1. [Amadeus Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) for covid-19 related information 
+2. [Amadeus Safety Place API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/safe-place) for Safety information
```

Comment:
Safety -> safety I think would be better

Reply:
i found that the name of API is 'Safe Place' not 'Safety Place' so I have changed to 'Safe Place' :)